sema/subyt/sinks.py

```
@@ -9,17 +9,18 @@
 log = logging.getLogger(__name__)
 
 
-def assert_writable(file_path: str, force_output: bool = False):
+def assert_writable(path_name: str | Path, force_output: bool = False):
```

Ruff (0.8.2)

Missing return type annotation for public function `assert_writable`

Add return type annotation: `None`

(ANN201)